aioshelly/block_device.py

```diff
@@ -195,11 +197,13 @@ def subscribe_updates(self, update_listener: Callable) -> None:
 
     async def update_status(self) -> None:
         """Device update from /status (HTTP)."""
-        self._status = await self.http_request("get", "status")
+        with contextlib.suppress(ClientResponseError):
```

Comment:
Is it not interesting for the library user to know that there was an error?

Reply:
`ClientResponseError` exception is raised here when initializing the battery powered device when the user has changed the password. Init happens on a task, so we are unable to catch this exception. In other cases, `settings` and `status` properties are protected and raise `NotInitialized` or `AuthRequired` exceptions.
https://github.com/home-assistant-libs/aioshelly/blob/b4ad8b776ffa99b8580645edafd460139f1dd3d2/aioshelly/block_device.py#L291-L311

Reply:
I agree with @MartinHjelmare we should at least log it as debug (we can't raise here since this code is called on a background task). We can also check the error code in the `ClientResponseError`.
However since this is a patch on v2 releases I would not change it now. We are moving to v3 soon (there is already a 3.0.0 release), we can improve this when adding this into the 3.x release.

Reply:
Sure, we can log this exception.